src/common/utils.js

```diff
@@ -18,10 +18,14 @@ module.exports = {
   map: null,
   mixin: null,
 
-  isMsie: function() {
+  isMsie: function(agentString) {
+    if (agentString === undefined) { agentString = navigator.userAgent; }
```

Comment:
ah, I didn't think about adding it like that, seems fine to me, I originally suggested to change the usage everywhere, but this is easier and makes sense too.